microprofile/testing/testing/src/main/java/io/helidon/microprofile/testing/HelidonTestConfigSynthetic.java

```diff
@@ -148,20 +147,24 @@ boolean useExisting() {
 
     private Config buildConfig() {
         List<ConfigSource> configSources = new ArrayList<>();
-        configSources.add(MpConfigSources.create(testInfo.id(), map));
+        ConfigSource configSource = MpConfigSources.create(testInfo.id(), map);
+        configSources.add(addConfigOrdinal(configSource, "1000"));
         blocks.forEach((type, values) -> {
             for (String value : values) {
-                configSources.add(MpConfigSources.create(type, new StringReader(value)));
+                ConfigSource config = MpConfigSources.create(type, new StringReader(value));
+                configSources.add(addConfigOrdinal(config, type, "900"));
             }
         });
         for (Method m : methods) {
-            configSources.add(invoke(ConfigSource.class, requireStatic(m), null));
```

Comment:
The `ConfigSource` is directly supplied by the user, and `ConfigSource` has an `getOrdinal()` which can have a custom implementation not based on the `config_ordinal` config key.

I.e., If a config source overrides the default implementation of `getOrdinal`, we shouldn't interfere.